code/modules/research/designs.dm

```diff
@@ -756,6 +756,13 @@ CIRCUITS BELOW
 	build_path = /obj/item/weapon/circuitboard/arcade/orion_trail
 	sort_string = "MAAAA"
 
+/datum/design/circuit/oriontrail

```

Comment:
Duplicate definition. Should be `datum/design/circuit/jukebox` or something along those lines.